docs/community/release_notes/release_note_0.5.0.md

```diff
@@ -0,0 +1,353 @@
+---
+hide:
+  - navigation
+
+license: |
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+      https://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+---
+
+# Apache Celeborn™ 0.5.0 Release Notes
+
+## Highlight
+- Celeborn support TLS for communication
+- Memory usage and performance optimization
+- Enhanced metrics and HTTP APIs
+- Support Spark3.5 with JDK21
```

Comment:
```suggestion
- Support Spark 3.5 with JDK 21
```

Reply:
actually, this is a little bit contrived, Spark 3.5 does not officially support Java 21

Reply:
Maybe just "Support Java 21"?